Nikita/PercolationStats.java

```diff
@@ -0,0 +1,67 @@
+import edu.princeton.cs.algs4.StdRandom;
+import edu.princeton.cs.algs4.StdOut;
+import edu.princeton.cs.algs4.StdStats;
+
+
+public class PercolationStats 
+{
+    private static final double CONF_VALUE = 1.96;
+    private final double[] fractionsOfSites;
+    private final int totalTries;
+    private final double meanRes;
+    private final double stddevRes;
+    
+    public PercolationStats(int n, int trials)
+    {   
+        validateInput(n, trials);
+        fractionsOfSites = new double[trials];
+        totalTries = trials;
+        int randomRow;
+        int randomCol;
+        int[] validInterval = {1, n+1};
```

Comment:
This variable is also not really necessary (in fact, it complicates reading somewhat)